tests/e2e-playwright/tests/sim4life.py

```diff
@@ -25,15 +30,25 @@ def on_web_socket(ws):
 
 
 @pytest.fixture
-def log_in_and_out(osparc_test_id_attribute: None, context: BrowserContext, page: Page):
+def api_request_context(context: BrowserContext):
```

Comment:
```suggestion
def api_request_context(context: BrowserContext) ->Iterator[Request]:
```
not sure about the type of request